### PR TITLE
Fixes skill level table in utils

### PR DESCRIPTION
**Pull request by Frankie-hz**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?
Added Rank G and the 70+ rank ranged for skill level calculations. These were missing for the last 8 years so why not add them now...right!?

These are currently only effecting magic hit rate.

TLDR: Mobs 70+ are missing a lot of MACC from the table being incomplete.

See: https://wiki.ffo.jp/html/2570.html for chart

## Steps to test these changes
Go to any mobskill that uses magic and see that you get resisted less? Not entirely sure what to put here :/

### CODE REVIEW

**Review by WinterSolstice8 — APPROVED** [review]

**Review by TeoTwawki — APPROVED** [review]
RubbadubdubSTAMP thanks for the post 75 age update. 🖖

File: scripts/globals/utils.lua [hunk 1]
```diff
@@ -348,15 +350,21 @@ local function getSkillLevelIndex(level)
         rangeId = 50
     elseif level > 60 and level <= 70 then
         rangeId = 60
-    else
+    elseif level > 70 and level <= 75 and (rank > 2) then -- If this is Rank A+ or A- then skip
```
Comment (claywar) [inline scripts/globals/utils.lua]: Useless parens here can be removed

Comment (Frankie-hz) [reply scripts/globals/utils.lua]: Fixed

**Review by claywar — APPROVED** [review]
